test/jessica_boss_web/models/user_model_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule JessicaBossWeb.UserModelTest do
+  use ExUnit.Case
+  use JessicaBossWeb.ModelCase
+
+  import Ecto.Query
+
+  alias JessicaBossWeb.UserModel
+  alias JessicaBoss.Repo
+
+  test "#insert inserts valid user" do
+    UserModel.insert(%{first_name: "Thiago", last_name: "Strutz", rfid: "Thiago-Strutz-rfid"})
```

Comment:
Always use bang methods (e.g., `Ecto.Repo.insert!/2`) if you're not going to use the returned value of a database operation. In this way, you can catch errors in the right spot. Otherwise, you won't know if the insertion failed and your test will have a weird behavior.